samples/stock/demo/basic-line/demo.js

```diff
@@ -19,6 +19,13 @@
             data: data,
             tooltip: {
                 valueDecimals: 2
+            },
+            lastPrice: {
+                enabled: true,
+                color: '#ff9d41',
```

Comment:
The color of the line should rather match the series' color. Could we additionally improve two things?
- let's add `xAxis.overscroll` to reserve some space for the label, maybe 1 week could be enough
- could you also style the label a bit? So it can remind a tiny version of the tooltip, like this:
![image](https://github.com/user-attachments/assets/9a7a50f5-8d42-4aee-98b0-e2ed2d13bcb0)